### ISSUE TITLE
Python 3.9, 3.10 support (install fails with `vatnumber`)

### ISSUE THREAD

**Issue by banagale** [django-getpaid/django-plans#148]
I've tried installing `django-plans` using python 3.9 and 3.10 using:

`pip install -e git://github.com/cypreess/django-plans.git#egg=django-plans`

The install fails at `vatnumber`, it looks like this library is no longer maintained. 

I wonder if `pyvat` would be a good drop-in replacement.

```
Collecting vatnumber>=1.2
  Using cached vatnumber-1.2.tar.gz (19 kB)
  Preparing metadata (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /Users/x/.virtualenvs/dj-plans-example39/bin/python3.9 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/51/z2bf1rq1007gxp1c2s6qvzym0000gn/T/pip-install-qp14ffg7/vatnumber_2dd164258b0242fdb0ad022a174a5bd3/setup.py'"'"'; __file__='"'"'/private/var/folders/51/z2bf1rq1007gxp1c2s6qvzym0000gn/T/pip-install-qp14ffg7/vatnumber_2dd164258b0242fdb0ad022a174a5bd3/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/51/z2bf1rq1007gxp1c2s6qvzym0000gn/T/pip-pip-egg-info-tv_x0e3n
       cwd: /private/var/folders/51/z2bf1rq1007gxp1c2s6qvzym0000gn/T/pip-install-qp14ffg7/vatnumber_2dd164258b0242fdb0ad022a174a5bd3/
  Complete output (1 lines):
  error in vatnumber setup command: use_2to3 is invalid.
```

**Comment by banagale** [comment]
It seems like maybe this was an issue due to trying to install the example project first.

**Comment by PetrDlouhy** [comment]
I realized, that `python-stdnum` would be better replacement for `vatnumber`. It is solved in #154
